obspy/taup/__init__.py

```diff
@@ -140,50 +140,109 @@
 
->>> arrivals = model.get_ray_paths(
-...     source_depth_in_km=500,
-...     distance_in_degree=130,
-...     phase_list=["Pdiff", "Sdiff", "pPdiff", "sSdiff"])
->>> arrivals.plot()  # doctest: +SKIP
+>>> arrivals = model.get_ray_paths(source_depth_in_km=500,
+...                                distance_in_degree=130,
+...                                phase_list=["Pdiff", "Sdiff",
+...                                            "pPdiff", "sSdiff"])
+>>> ax = arrivals.plot_rays()
 
 .. plot::
     :width: 50%
     :align: center
 
     from obspy.taup import TauPyModel
-    TauPyModel().get_ray_paths(
-        500, 130,
-        phase_list=["Pdiff", "Sdiff", "pPdiff", "sSdiff"]).plot()
+    TauPyModel().get_ray_paths(500, 130,
+                               phase_list=["Pdiff", "Sdiff", "pPdiff",
+                                           "sSdiff"]).plot_rays()
 
 Additionally, Cartesian coordinates may be used instead of a polar grid:
 
 >>> arrivals = model.get_ray_paths(source_depth_in_km=500,
 ...                                distance_in_degree=130,
 ...                                phase_list=["ttbasic"])
->>> arrivals.plot(plot_type="cartesian")  # doctest: +SKIP
+>>> ax = arrivals.plot_rays(plot_type="cartesian")
 
 .. plot::
     :width: 75%
     :align: center
 
     from obspy.taup import TauPyModel
-    TauPyModel().get_ray_paths(
-        500, 130, phase_list=["ttbasic"]).plot(plot_type="cartesian")
+    TauPyModel().get_ray_paths(500, 130,
+                               phase_list=["ttbasic"]).plot_rays(plot_type=
+                                                                 "cartesian")
+
+Travel times for these ray paths can be plotted using the
+:meth:`~obspy.taup.tau.Arrivals.plot_times` method:
+
+>>> arrivals = model.get_ray_paths(source_depth_in_km=500,
+...                                distance_in_degree=130)
+>>> ax = arrivals.plot_times()
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import TauPyModel
+    ax = TauPyModel().get_ray_paths(500, 130).plot_times()
+
+Alternatively, convenience wrapper functions plot the arrival times
+and the ray paths for a range of epicentral distances.
+
+The travel times wrapper function is:
+
+>>> from obspy.taup import plot_travel_times
+>>> import matplotlib.pyplot as plt
+>>> fig, ax = plt.subplots()
+>>> ax = plot_travel_times(source_depth=10, phase_list=["P","S","PP"],
+...                        ax=ax, fig=fig)
+There were 2 epicentral distances without an arrival
+
+.. plot::
+    :width: 50%
+    :align: center
+
+    from obspy.taup import plot_travel_times
+    import matplotlib.pyplot as plt
+
+    fig, ax = plt.subplots()
+    ax = plot_travel_times(source_depth=10, ax=ax, phase_list=["P","S","PP"],
+    ...                    fig=fig)
+
+The ray path plot wrapper function is:
+
+>>> from obspy.taup import plot_ray_paths
+>>> import matplotlib.pyplot as plt
+>>> ax = plt.subplot(111, polar=True)
+>>> fig = ax.figure
```

Comment:
please compress this as `fig, ax = plt.subplots(...)` like above and to be consistent across all examples

Reply:
done